test/module/irohad/torii/processor/transaction_processor_test.cpp

```diff
@@ -103,14 +122,8 @@ class TransactionProcessorTest : public ::testing::Test {
 TEST_F(TransactionProcessorTest, TransactionProcessorOnProposalTest) {
   std::vector<shared_model::proto::Transaction> txs;
   for (size_t i = 0; i < proposal_size; i++) {
-    auto &&tx = shared_model::proto::TransactionBuilder()
-                    .createdTime(iroha::time::now())
-                    .creatorAccountId("admin@ru")
-                    .addAssetQuantity("admin@tu", "coin#coin", "1.0")
-                    .build()
-                    .signAndAddSignature(
-                        shared_model::crypto::DefaultCryptoAlgorithmType::
-                            generateKeypair());
+    auto &&tx = base_tx().build().signAndAddSignature(
```

Comment:
What is the point of && here?

Reply:
I mean, what is the reason of it?

Reply:
idk, it wasn't introduced here
look [here](https://github.com/hyperledger/iroha/commit/865e6e5889b38d75eb8d469cdbaf25cd59f66340#diff-138716dd00810329bf6f77447981241fR103)